schema-guide.md

```diff
@@ -404,26 +412,26 @@ primary principle, "Importance", when others cite this work.
 - **usage**:<br><br>
     ```yaml
     references:
-      - type: software
-        authors:
+      - authors:
           - name: "The Dependency Project"
-        title: "Dependency"
-        date-released: 2021-07-26
+        date-released: "2021-07-26"
         doi: 10.5281/zenodo.x1234567
-        version: 0.13.4
         repository-code: "https://github.com/dependency-project/dependency"
-      - type: article
-        scope: Cite this paper if you want to reference the general concepts of the software.
-        authors:
+        title: Dependency
+        type: software
+        version: "0.13.4"
```

Comment:
Do we treat these consistently? semver doesn't need to be quoted, but it's very close to being interpreted as a number

Reply:
We don't treat the consistently (yet). E.g., `cff-version` is always unquoted, `version` is mixed. 
Now that we have the text snippet about string quoting included, it should be safe to leave them all unquoted, although yes, they are close to number-like data. It's not something that we can solve, and I think it's better to have them in onw way or the other, and I think we decided against quoting everything?

Reply:
Resolved in a997ed6 (see commit message for details)